inno/insecure-cookie.yaml

```
@@ -0,0 +1,24 @@
+rules:
+- id: insecure-cookie
+  patterns:
+  - pattern-either:
+    - pattern: r.set_cookie(..., secure=False)
+    - pattern: r.set_cookie(..., secure=False, ...)
```

```suggestion
    - pattern: r.set_cookie(..., secure=False, ...)
```

The ellipses `...` can capture something or nothing. So the first pattern here is redundant.